lib/commands.js

```diff
@@ -547,10 +548,13 @@ exports.bindCommands = function bindCommands(forum) {
                 if (!checkAvailable(forbiddenCmds, cmd, 'error', forbiddenMsg)) {
                     return reject(new Error('E_FORBIDDEN_COMMAND'));
                 }
+                if (context === undefined) {
+                    context = new Object();
```

Comment:
on its own, no. I got the syntax wrong in the initial commit, but i corrected that in the comments for 617406a

i could have rewritten the history and pushed a new branch.... but i chickened out on that becuase rewriting history is scary.